src/ase/cw/model/OrderConsumer.java

```diff
@@ -5,10 +5,12 @@
  */
 public interface OrderConsumer {
 
-    void setName(String time);
+    void setName(String name);
+    void setBusy(boolean busy);
```

Comment:
mentiont previously:
if you want a method like  boolean isBusy(); (which is ok) this does not mean you need a method void setBusy(boolean busy);
the server knows when he is busy, so the server/orderconsumer can change the busy state itself